data-prepper-core/src/main/java/com/amazon/dataprepper/DataPrepperExecute.java

```diff
@@ -10,6 +10,9 @@
     private static final Logger LOG = LoggerFactory.getLogger(DataPrepperExecute.class);
 
     public static void main(String[] args) {
+        if(args.length > 1) {
```

Comment:
I feel this way of ordered args is not user friendly, we can fix this later. Can we make sure we actually add this to readme and add a comment here as well.

Reply:
Good call out, i forgot to update the readme